.gitlab-ci.yml

```diff
@@ -1156,13 +1156,22 @@ workflow:
       # Pull coverage cache on dev branches
       COVERAGE_CACHE_FLAG: "--pull-coverage-cache"
 
+.on_gitlab_changes_success:
+  changes:
+    paths:
+      - .gitlab-ci.yml
+      - .gitlab/**/*
+      - .gitlab/**/.*
+    compare_to: main # TODO: use a variable, when this is supported https://gitlab.com/gitlab-org/gitlab/-/issues/369916
+
 .on_gitlab_changes:
   - !reference [.except_mergequeue]
```

Comment:
What about you just remove `.except_mergequeue` from that rule. And use directly `.except_mergequeue` in jobs that need it.
It starts to be a bit hard to see the difference between `.on_gitlab_changes_success` and `.on_gitlab_changes` when reading the name of the rules